### ISSUE TITLE
Error after installing: Registry key not found

### ISSUE THREAD

**Issue by julmot** [crazy-max/nodejs-portable#12]
After installing NodeJS with this batch and the correct settings on line 32 of the batch I get the following error message in the console if I launch `1 Launch`:
```javascript
ERROR: The specified registry key or value was not found.
Your environment has been set up for using Node.js 0.10.29 (x64) and npm.
```

**Comment by crazy-max** [comment]
If Git is not installed (with an installer) on your computer you get this error.
I will fixed it for the next release

**Comment by crazy-max** [comment]
Solved on v1.11
